docs/dal/index.rst

```diff
@@ -288,8 +315,38 @@ Available values:
 
 This service exposes the :ref:`verbosity <pyvo-verbosity>` parameter
 
+.. _pyvo-ssa:
+
 Simple Spectrum Access
 ----------------------
+The `Simple Spectral Access (SSA) Protocol (SSAP) <https://www.ivoa.net/documents/SSA/>`_ 
+defines a uniform interface to remotely discover and access one 
+dimensional spectra. SSA is a member of an integrated family of data access 
+altogether comprising the Data Access Layer (DAL) of the IVOA. 
+SSA is based on a more general data model capable of describing most tabular 
+spectrophotometric data, including time series and spectral energy distributions 
+(SEDs) as well as 1-D spectra; however the scope of the SSA interface as 
+specified in this document is limited to simple 1-D spectra, including 
+simple aggregations of 1-D spectra. The form of the SSA interface is simple: 
+clients first query the global resource registry to find services of interest 
+and then issue a data discovery query to selected services to determine what 
+relevant data is available from each service; the candidate datasets 
+available are described uniformly in a VOTable format document which is 
+returned in response to the query. Finally, the client may retrieve selected 
+datasets for analysis. Spectrum datasets returned by an SSA spectrum service 
+may be either precomputed, archival datasets, or they may be virtual data 
+which is computed on the fly to respond to a client request. 
+Spectrum datasets may conform to a standard data model defined by SSA, 
+or may be native spectra with custom project-defined content. 
+Spectra may be returned in any of a number of standard data formats. 
+Spectral data is generally stored externally to the VO in a format specific 
+to each spectral data collection; currently there is no standard way to 
+represent astronomical spectra, and virtually every project does it 
+differently. Hence spectra may be actively mediated to the standard 
+SSA-defined data model at access time by the service, so that client analysis 
+programs do not have to be familiar with the idiosyncratic details of 
+each data collection to be accessed.
+
```

Comment:
This might be a bit too long and detailed as compared with the other ones.

Reply:
Thank you very much for your review, I made a couple of changes to the texts according to your suggestions. Please have a look if they are appropriate now.